### PR TITLE
Add Regions blog post

### PR DESCRIPTION
**Pull request by FloorD**
Adds Regions blog post.

- Thanks @sjmiller609 for the first feedback
- Needs header image (already requested)

### CODE REVIEW

**Comment by FloorD** [comment]
Might need some input on what to use for the code block etc.

**Review by DarrenBaldwin07 — APPROVED** [review]

**Review by sjmiller609 — APPROVED** [review]

File: src/content/blog/2024-08-12-aws-regions/index.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## Deploying to Regions
+
+When creating an Instance, pick a Region using the drop-down selector. 
+
+![Pick a cloud region via the Tembo Cloud UI](./region-dropdown.png)
+
+After deploying an Instance, the Region cannot be changed.
+
+### API
+When using the [Tembo API](https://github.com/tembo-io/website/blob/725f4104f33f45fd18fb395a3a0b16fc4f4fedf2/docs/development/api), Providers and Regions are represented as Dataplanes. Dataplanes are where Instances are hosted. A Dataplane is uniquely identified by `provider_id`, `region_id`, and `index`.
```
Comment (sjmiller609) [inline src/content/blog/2024-08-12-aws-regions/index.mdx:27]:
```suggestion
When using the [Tembo API](/docs/development/api), Providers and Regions are represented as Dataplanes. Dataplanes are where Instances are hosted. A Dataplane is uniquely identified by `provider_id`, `region_id`, and `index`.

```

File: src/content/blog/2024-08-12-aws-regions/index.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "provider_name": "AWS",
+    "region": "us-east-1",
+    "region_id": "use1",
+    "region_name": "US East 1 (N. Virginia)"
+  }
+```
+
+When you create an Instance, you may specify the Provider and Region using `provider_id` and `region_id`. Tembo decides which index to deploy your Instance into. By default, instances are deployed to `aws`, `use1`.
+
+Some queries to Tembo's APIs involve fetching private data from your Instance, for example [Logs](https://github.com/tembo-io/website/blob/725f4104f33f45fd18fb395a3a0b16fc4f4fedf2/docs/product/cloud/troubleshooting/logs), [Metrics](https://github.com/tembo-io/website/blob/725f4104f33f45fd18fb395a3a0b16fc4f4fedf2/docs/product/cloud/troubleshooting/metrics), and [Secrets](https://github.com/tembo-io/website/blob/725f4104f33f45fd18fb395a3a0b16fc4f4fedf2/docs/product/cloud/security/update-postgres-password). These queries go to the [Dataplane API](https://api.data-1.use1.tembo.io/redoc), which has a different domain name for each Dataplane. This model allows for more secure handling of sensitive information.
```
Comment (sjmiller609) [inline src/content/blog/2024-08-12-aws-regions/index.mdx:42]:
```suggestion
Some queries to Tembo's APIs involve fetching private data from your Instance, for example [Logs](/docs/product/cloud/troubleshooting/logs), [Metrics](/docs/product/cloud/troubleshooting/metrics), and [Secrets](/docs/product/cloud/security/update-postgres-password). These queries go to the [Dataplane API](https://api.data-1.use1.tembo.io/redoc), which has a different domain name for each Dataplane. This model allows for more secure handling of sensitive information.

```

**Review by sjmiller609 — APPROVED** [review]

**Comment by samay-sharma** [comment]
@FloorD : This is not passing build. Can you rebase on main and then make sure build passes. Then, I can approve.

**Comment by FloorD** [comment]
> Merging can be performed automatically with 1 approving review.

Done!

**Review by samay-sharma — APPROVED** [review]

**Pull request merged**